processor/src/main/java/io/jonasg/bob/definitions/TypeDefinition.java

```diff
@@ -58,12 +59,15 @@ public List<SetterMethodDefinition> getSetterMethods() {
 		for (FieldDefinition field : fields) {
 			String name = field.name().substring(0, 1).toUpperCase() + field.name().substring(1);
 			methodsWithOneParam.stream()
-					.filter(m -> m.name().equals(field.name()))
+					.filter(m -> m.name().equals(field.name()) || (m.name().equals(field.name().substring(2))
```

Comment:
I would like to see a stronger check by first verifying if the field name starts with "is" and afterwards check it for equality by stripping the first two chars.